reviews.Rmd

```diff
@@ -0,0 +1,583 @@
+Faced with a specific tradeoff, e.g., is a particular post hoc change of a preregistration beneficial, we must revert to vague notions and "gut" instinct under this philosophy.
+Within the Bayesian framework, we may directly ask: What consequences should preregistration have on the beliefs of individual scientists or the research community as a whole?
+
+:::reviewer
+The title should not read a Bayesian rationalization, but a likelihood rationalization.
+A higher likelihood is not always better, as one can cherry pick a high likelihood ratio.
+This is extensively discussed by Mayo (2018).
+After p-hacking, we could get a very high likelihood (or Bayes factor) that supports H1 - but it was cherry-picked, and is highly likely to be a fluke.
+Therefore, the authors limit themselves to relative likelihoods.
+:::
+
+We did not limit ourselves to relative liklihoods.
+We explicitly include both the "firmness" and "increase in firmness" perspective.
+That a firmness perspective has significant flaws is of no matter to our argument since we intend to show that someone who subscribes to either must be more persuaded by a preregistered study.
+
+:::reviewer
+They point out that, assuming people only increase their alpha level when p-hacking, preregistration leads to tests with a lower alpha level.
+As such, they basically restate Mayo's (2018) argument about the severity of a test.
+:::
+
+Let us cite <https://doi.org/10.3758/s13423-022-02069-1"Nowhere in Mayo’s severity principle or in the specifications discussed below is the riskiness or specificity of a hypothesis, and the predictions it makes, incorporated as a requirement for a severe test (Meehl, 1978; 1986; 1990b, a; Roberts & Pashler, 2000)."
+
+:::reviewer
+However, they never make this explicit, and I think this is to the detriment of the paper.
+:::
+
+We added a new section that deals with the concept of severity, making it explicit how our approach differs fundamentally.
+
+:::reviewer
+The fact that they subsequently add a Bayesian sauce to the likelihood ratio (by adding a distribution to the Type 1 error rate when a study is not preregistered, instead of simply making some assumption about a point-value of the inflated Type 1 error rate) does not add anything to their argument.
+So, I believe the authors have basically attempted to re-invent Mayo's severity argument.
+:::
+
+The whole second part of the paper deals with the problem of not being able to reduce the theoretical risk to a point estimate.
+The statistical procedure has an objective Type 1 error rate, for which we can come up with a point estimate if we agree on the conditions, which is a starting point for evaluating the theoretical risk of the scientific (!= statistical) procedure.
+Since, a reader does not have access to the whole scientific procedure, they must judge it with uncertainty.
+We find it important to distentengle the statistical procedure, the scientific procedure, and the outside perspective of the reader.
```

Comment:
I wonder if that would also be his review on Bayesian Textbooks: "Why don't they just use a point estimate, no need for this bayesian sauce" ^^. Adding a distribution to quantify uncertainty is just the correct mathematical formulation. Moving around point estimates is not.
```suggestion
Since readers do not have access to the whole scientific procedure, they must judge it with uncertainty.
We find it important to disentangle the statistical procedure, the scientific procedure, and the outside perspective of the reader.
```